config/jobs/kubernetes-sigs/wg-multi-tenancy/hnc-postsubmit.yaml

```diff
@@ -0,0 +1,14 @@
+postsubmits:
+  kubernetes-sigs/multi-tenancy:
+  - name: pull-hnc-test
+    annotations:
+      testgrid-dashboards: wg-multi-tenancy-hnc
+      testgrid-tab-name: postsubmit-tests
+    decorate: true
+    path_alias: sigs.k8s.io/multi-tenancy
+    run_if_changed: "incubator/hnc/.*"
+    spec:
+      containers:
+      - image: golang:1.14
+        command:
+        - ./incubator/hnc/hack/ci-test.sh
```

Comment:
We should fork this file before merging this PR so we don't need to modify `test-infra` when our presubmits and postsubmits diverge (which they should very soon).

/hold

Reply:
done

Reply:
Well, done once https://github.com/kubernetes-sigs/multi-tenancy/pull/1080 is merged, right? :)